packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -1273,6 +1275,130 @@ describe('shallow', () => {
         expect(wrapper.find(Component.displayName)).to.have.lengthOf(2);
       });
     });
+
+    describeWithDOM('find DOM elements by constructor', () => {
+
+      it('finds DOM elements and attributes by constructor', () => {
+        /* eslint-env browser */
+        const map = {
```

Comment:
now that this map lives in two places, let's extract it out into a single file, and import it in both test files?